src/OutputWriters/netcdf_output_writer.jl

```diff
@@ -32,53 +33,134 @@ mutable struct NetCDFOutputWriter{D, O, I, F, S} <: AbstractOutputWriter
       clobber :: Bool
        slices :: S
      previous :: Float64
+      verbose :: Bool
 end
 
 """
-    NetCDFOutputWriter(model, outputs; frequency=nothing, interval=nothing, filename=".",
+    NetCDFOutputWriter(model, outputs; filename, frequency=nothing, interval=nothing,
                        global_attributes=Dict(), output_attributes=Dict(), dimensions=Dict(),
-                       clobber=true, compression=0, slice_kw...)
+                       clobber=true, compression=0, include_halos=false, verbose=false, slice_kwargs...)
```

Comment:
This keyword argument could also be `with_halos`. We have a function `with_tracers` (this might be the only `with_*` currently); I had imagined we might have more `with_*` in the futures (eg a `with_boundary_conditions` function for `Fields`. So we could use that kind of nomenclature here.

Reply:
Good point. Done.